tests/bwrite.rs

```diff
@@ -17,128 +17,554 @@
+        "map-field": {"l0": <<1, 2>>, "l1": <<"str1", "str2">>, "l2": true},
+        "binary-field": b"\x00",
+        "binary-set-field": <<b"\x01", b"\x02">>}"#,
+    ])
+    .output()?;
+
     let mut c = tm.command()?;
     let get_cmd = c.args(&[
         "--region",
         "local",
         "--table",
         &table_name,
         "get",
-        "ichi",
+        "11",
         "-o",
         "raw",
     ]);
-
     let output = get_cmd.output()?.stdout;
     let data: Value = serde_json::from_str(&String::from_utf8(output)?)?;
+    assert_eq!(data["pk"]["S"], "11");
+    assert_eq!(data["null-field"]["NULL"], true);
+    assert_eq!(data["list-field"]["L"][0]["N"], "1");
+    assert_eq!(data["list-field"]["L"][1]["N"], "2");
+    assert_eq!(data["list-field"]["L"][2]["N"], "3");
+    assert_eq!(data["list-field"]["L"][3]["S"], "str");
+    assert_eq!(data["map-field"]["M"]["l0"]["NS"][0], "1");
+    assert_eq!(data["map-field"]["M"]["l0"]["NS"][1], "2");
+    assert_eq!(data["map-field"]["M"]["l1"]["SS"][0], "str1");
+    assert_eq!(data["map-field"]["M"]["l1"]["SS"][1], "str2");
+    assert_eq!(data["map-field"]["M"]["l2"]["BOOL"], true);
+    assert_eq!(
+        data["binary-field"]["B"],
+        general_purpose::STANDARD.encode(b"\x00")
+    );
+    assert_eq!(
+        data["binary-set-field"]["BS"][0],
+        general_purpose::STANDARD.encode(b"\x01")
+    );
+    assert_eq!(
+        data["binary-set-field"]["BS"][1],
+        general_purpose::STANDARD.encode(b"\x02")
+    );
 
-    let binary = data["Binary"]["B"].as_str().unwrap();
-    assert_eq!(binary, "dGhpcyB0ZXh0IGlzIGJhc2U2NC1lbmNvZGVk");
+    Ok(())
+}
 
-    // The order of the values within a set is not preserved, so I will use HashSet to compare them.
-    let binary_set: HashSet<String> = data["BinarySet"]["BS"]
-        .as_array()
-        .unwrap()
-        .iter()
-        .filter_map(Value::as_str)
-        .map(|s| s.to_string())
-        .collect();
-    let binary_set_expected: HashSet<String> =
-        HashSet::from(["U3Vubnk=", "UmFpbnk=", "U25vd3k="].map(String::from));
-    assert_eq!(binary_set, binary_set_expected);
+#[tokio::test]
+async fn test_batch_write_put_sk() -> Result<(), Box<dyn std::error::Error>> {
+    let mut tm = util::setup().await?;
+    let table_name = tm.create_temporary_table("pk", Some("sk")).await?;
+
+    let mut c = tm.command()?;
+    c.args(&[
+        "--region",
+        "local",
+        "--table",
+        &table_name,
+        "bwrite",
+        "--put",
+        r#"{"pk": "11", "sk": "111"}"#,
+    ])
+    .output()?;
+
+    let mut c = tm.command()?;
+    let get_cmd = c.args(&[
+        "--region",
+        "local",
+        "--table",
+        &table_name,
+        "get",
+        "11",
+        "111",
+        "-o",
+        "json",
+    ]);
+    get_cmd.assert().success().stdout(
+        predicate::str::is_match(r#"pk": "11""#)?.and(predicate::str::is_match(r#"sk": "111""#)?),
```

Comment:
I believe it must be `r#""pk": "11""#` and `r#""sk": "111""#`.

Reply:
Fixed.